packages/tools/guiEditor/src/components/commandBarComponent.tsx

```diff
@@ -140,6 +140,16 @@ export class CommandBarComponent extends React.Component<ICommandBarComponentPro
                                         this.props.globalState.onSnippetLoadObservable.notifyObservers();
                                     },
                                 },
+                                {
+                                    label: "Save selected contorl",
+                                    onClick: () => {
+                                        this.props.globalState.onSaveSelectedControl.notifyObservers();
+                                    },
+                                },
+                                {
+                                    label: "Load contorl",

```

Comment:
nit: contorl -> control

Reply:
spelling fix is added in the new commit

commit: ef8410cf2835f23dd355f8baf525218c9a4bca67